content/en/Platform Deep Dive/Assets/risk-advisories.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Explore NVD Risk Advisories"
```

Comment:
Due to a late change in the API due to performance concerns, we've split the technical implementation. The API to retrieve CVEs is no longer through NVD, but through an EU based entity (circl.lu). We are still showing NVD links to web-based CVEs though.

Because of this, we've been downplaying the 'NVD' part of the name, and focusing on either just 'Risk Advisories' or using variants like 'CVE-based Risk Advisories'